packages/web/src/app/orbs/[power]/page.tsx

```diff
@@ -0,0 +1,70 @@
+import { notFound } from "next/navigation";
+
+interface PowerDetails {
+  name: string;
+  description: string;
+  strategy: string;
+  memberOnly?: boolean;
+}
+
+const powerDetails: Record<string, PowerDetails> = {
+  multiply: {
+    name: "Multiply",
+    description:
+      "Generates a new piece in the arena. This new piece will have no powers of its own, but by adding pieces, you gain more coverage and control of the battle arena and limit your opponent&apos;s movement.",
+    strategy:
+      "Use it to confront an opponent and entice them to jump your newly formed piece. Then jump them back, and you come out of the confrontation ahead.",
+  },
+  "cancel-multiply": {
+    name: "Cancel Multiply",
+    description:
+      "Removes a multiplied piece from the board. This power only works on pieces that were created using the Multiply power. It cannot remove original pieces.",
```

Comment:
Incorrect - as per manual:
```
If you changed your mind about multiplying this piece, or realized you can't move this new piece because it is surrounded by off-limit tiles, you can cancel your choice and choose another strategy.
Strategy: This is not a power acquired from the Orbs, but rather a way to change your mind. Use it to choose a different, last minute decision. 
```